### DISCUSSION TITLE
TailwindCSS VM support

### DISCUSSION THREAD

**Discussion by petarvujovic98** [NearSocial/viewer#119] [General]
What do you think about adding TailwindCSS as a supported CSS framework?
I think that it is one of the most popular libraries and many developers are transitioning to using it for front-end development.

**P.S.** If you do want to support it I could lend a hand in working on PRs for this.

**Comment by evgenykuzyakov** [comment]
Previously discussed the CSS frameworks. The best option is StyledComponents instead of some global collection of classes. Bootstrap is likely to be deprecated at some point once majority of the design is replaced with styled components and there are libraries to use and style them.
